ci_build/view_log.py

```diff
@@ -0,0 +1,91 @@
+import re
+import json
+import os
+import sys
+
+from django.views.generic import TemplateView
+
+from community.views import get_header_and_footer
+from community.git import (get_org_name,
+                           get_owner,
+                           get_deploy_url,
+                           get_upstream_deploy_url)
+
+
+class BuildLogsView(TemplateView):
+    template_name = 'build_logs.html'
+
+    def get_build_info(self):
+        data = {
+            'Org name': get_org_name(),
+            'Owner': get_owner(),
+            'Deploy URL': get_deploy_url(),
+        }
+        try:
+            upstream_deploy_url = get_upstream_deploy_url()
+            data['Upstream deploy URL'] = upstream_deploy_url
+        except RuntimeError:
+            data['Upstream deploy URL'] = 'Not found'
+        return dict(data)
+
+    def get_build_logs(self):
+        log_lines = []
+        log_level_specific_lines = {
+            'INFO': [],
+            'DEBUG': [],
+            'WARNING': [],
+            'ERROR': [],
+            'CRITICAL': []
+        }
+        log_file_path = './_site/community.log'
+        log_file_exists = os.path.isfile(log_file_path)
+        if log_file_exists:
+            with open(log_file_path) as log_file:
```

Comment:
I'm doing this because the message in html page will shown accordingly. If the file doesn't exists a different message will be displayed asking user to run the build command. https://github.com/coala/community/pull/251/commits/31f21359d68f8b6b1d17d623f3ef014289e523b0#diff-b640fac2a47fbe8bce3f0bfc4a528452R67

Reply:
Our README.md should explain how to setup the project and how to run it.

Reply:
Yeah! It should. Already created a task in phase-3 where all the required information will updated in repositories README’s as well as docs.

Reply:
> You should directly do `with open('./_site/community.log') as log_file:` as it was done earlier without above three lines.

So, I think we can go ahead with the above solution? Any way if the file doesn't exist the other commands will fail, right?

> I'm doing this because the message in html page will shown accordingly. If the file doesn't exists a different message will be displayed asking user to run the build command. [31f2135#diff-b640fac2a47fbe8bce3f0bfc4a528452R67](https://github.com/coala/community/commit/31f21359d68f8b6b1d17d623f3ef014289e523b0#diff-b640fac2a47fbe8bce3f0bfc4a528452R67)

Reply:
Yeah! They’ll fail to get executed